### PR TITLE
higher batch size details.

### PR DESCRIPTION
**Pull request by sayakpaul**
FYI @jerryzh168 @drisspg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,13 +144,57 @@ With the newly added `fp8dqrow` scheme, we can bring down the inference latency
...
+</details>
+
+But interestingly, if we use an exotic fpx scheme for quantization, we can afford lesser memory with an increase in the latency:
+
+<details>
+<summary>Collapse table</summary>
+
+|    | ckpt_id                      |   batch_size | fuse   | compile   | compile_vae   | quantization   | sparsify   |   model_memory |   inference_memory |   time |
+|---:|:-----------------------------|-------------:|:-------|:----------|:--------------|:---------------|:-----------|---------------:|-------------------:|-------:|
+|  0 | black-forest-labs/FLUX.1-dev |           16 | False  | True      | True          | fp6_e3m2       | False      |         17.591 |             49.938 | 61.649 |
```
Comment (jerryzh168) [inline README.md:181]: maybe we could optimize the time further

File: README.md [hunk 2]
```diff
@@ -144,13 +144,57 @@ With the newly added `fp8dqrow` scheme, we can bring down the inference latency
...
 ### Trade-offs, trade-offs, and more trade-offs
 
 We know that the table included above is hard to parse. So, we wanted to include a couple of points that are worth noting. 
 
 * Select the quantization technique that gives you the best trade-off between memory and latency. 
 * A quantization technique may exhibit different optimal settings for a given batch size. For example, for a batch size of 4, `int8dq` gives best time without any QKV fusion. But for other batch sizes, that is not the case.
 
+The section below, drives this point home.
+
+#### Higher batch sizes like 16 and above
```
Comment (jerryzh168) [inline README.md:157]: does integer quant gives speedup or is it mostly slow down right now? like int4wo, int8wo, int8dyn?

Comment (sayakpaul) [reply README.md:157]: It does but FP is just faster at the moment.

File: README.md (hunk 1 above)
Comment (sayakpaul) [reply README.md:181]: Do you have any ideas on how?

**Comment by sayakpaul** [comment]
@a-r-r-o-w a gentle ping.

**Review by a-r-r-o-w — APPROVED** [review]
Thank you, looks awesome! The speed/memory aligns with the observations for CogVideoX
